platform/mock-local-home-platform.ts

```diff
@@ -38,28 +40,65 @@ export class UDPScanConfig implements ScanConfig {
   }
 }
 
-// TODO(cjdaly): figure out how to direct arbitrary instances of stub classes to an arbitrary
-// instance of MockLocalHomePlatform. May need to use some static event or singleton pattern.
-
 // TODO(cjdaly): add other radio scan support
 export class MockLocalHomePlatform implements MockUDPListener {
+  //  Singleton instance
+  private static instance: MockLocalHomePlatform;
+
   private udpScanConfigs: UDPScanConfig[] = [];
-  private network: MockNetwork;
+  private mockNetwork: MockNetwork;
+  private app: AppStub;
+  private deviceManager: smarthome.DeviceManager;
   private localDeviceIds: string[] = [];
```

Comment:
this should probably be a map (the platform should store the association between deviceId and localDeviceId)

Reply:
will add

Reply:
Done